src/components/Project/AragonClient.js

```diff
@@ -324,12 +336,12 @@ class AragonClient extends React.Component {
                   <div className="badge-container">
                     <AutarkBadge>Autark</AutarkBadge>
                   </div>
-                  <h3>Reputation Support</h3>
+                  <h3>Reputation</h3>
                   <h4>
-                    Expand governance possibilities by providing more features
+                    Expand governance possibilities by providing features
                     for reputation-based organizations, including the ability to
-                    allocation non-transferable tokens in place of (or in
-                    addition to) traditional bounties.
+                    allocate non-transferable tokens instead of (or in
+                    addition to) regular bounties to compensate labor.
```

Comment:
> allocate non-transferable tokens instead of (or in addition to) regular bounties to compensate labor.

I think "monetary bounties" would be more clear than "regular bounties".